config/environments/development.rb

```diff
@@ -67,4 +67,15 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = { host: ENV["HOST"]}
+  config.action_mailer.smtp_settings = {
+    :user_name => ENV["EMAIL_USER_NAME"],
```

Comment:
ửa tương tự bên dưới
```suggestion
    user_name: ENV["EMAIL_USER_NAME"],
```

Reply:
send cho phép gọi phương thức của một đối tuongj dưới dạng string hoặc symbol, send có thể gọi đc cả hàm private khi ở ngoài class ạ.